packages/build/test/integration/utils.test.js

```diff
@@ -0,0 +1,60 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback-next
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+const assert = require('assert');
+const path = require('path');
+const fs = require('fs');
+
+const utils = require('../../bin/utils');
```

Comment:
We don't really use `utils` in this test case. Can we test its exported functions here as well as the scripts in `/bin`? You can test them in separate files or describe blocks, but I leave the organization up to you.

Reply:
Good catch. Fixed.